client/src/Organisms/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,56 @@
+import { BsTrophy, BsTrophyFill } from 'react-icons/bs';
+import { FaDog, FaUserCircle, FaUsersCog } from 'react-icons/fa';
+import { useContext } from 'react';
+
+import MenuItem from '../../Molecules/MenuItem/MenuItem';
+import SidebarItemsStyled from './SidebarItemsStyled';
+import SidebarStyled from './SidebarStyled';
+import { UserDataContext } from './../../Context/UserDataContext';
+
+const Sidebar = () => {
+  const { state } = useContext(UserDataContext);
+  const { isAuthenticated } = state;
+  const { userId } = state;
+  const fontsize = '30px';
+
+  if (isAuthenticated) {
```

Comment:
You could do "early return" pattern here, so switch the order. If(!isAuthenticated?){ render null} else {rest of it} or you could try to write return ({isAuthenticated? (all stuff to render):null}). Just sother solutions, no need to refactor

Reply:
Interesting...